Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -375,6 +375,53 @@ def translate_severity(alert):
         return 1
     return 0
 
+def get_RQL_response():
+
+    """"
+    Retrieve any RQL
+    """
+    RQL = demisto.getArg('rql').encode("utf-8")
+
+    payload = {"query":RQL, "filter": {}}
+
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+
+    if not RQL:
+        return_error('You must specify an RQL to retrieve the response')
+    response = req('POST', 'search/config', payload, None)
+    context = {}
+    #MD = tableToMarkdown("RQLoutput", response)
+    #TODO add check here that data exists
+
+    MD2 = tableToMarkdown("RQLoutput", response["data"]["items"])
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': response,
+        'EntryContext': {'Redlock':response},
+        'HumanReadable': MD2
+    })
+
+
+def remediate_alerts():
+    """
+    Retrieve remediation details for a given alert
+    """
+    alert_ids = argToList(demisto.getArg('alert-id'))
+    payload = {'alerts': alert_ids, 'filter': {}}
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+
+    if not alert_ids:
+        return_error('You must specify the alert-id to retrieve with CLI remediation')
+
+    md_data = []
+    context = []
+    response = req('PATCH', 'alert/remediation/'+ demisto.getArg('alert-id'), None, None)
+
+    demisto.results(str(response))
```

Comment:
add a full entry, md, context, raw response....
```suggestion
    demisto.results(str(response))
```